docs/guides/data.md

```diff
@@ -0,0 +1,62 @@
+# Obtaining Data
+
+Many operations you might want to do via your bot may require additional data to complete, for example, information
+about users, guilds, channels, and so on. There's two main ways of obtaining this information, which is detailed below.
+
+## REST
+
+The first option is to query the Discord API directly for this information, which, while typically guarantees to
+provide the most up-to-date information possible, is much slower and consumes API ratelimits. Therefore it is advisable
+to avoid calling these often (e.g. per event).
+
+```py
+bot = hikari.GatewayBot(...)
+# OR
+bot = hikari.RESTBot(...)
+
+# -- Snip --
+
+# Requesting a specific guild's data
+guild_id = ...
+guild = await bot.rest.fetch_guild(guild_id)
+print(f"The guild's name is {guild.name}!")
+
+```
+
+All rest calls can be performed via the bot's `RESTClient` instance, which can be accessed via `bot.rest`.
```

Comment:
```suggestion
All rest calls can be performed via the `bot`'s `RESTClient` instance, which can be accessed via `bot.rest`.
```
because you're referring to the `bot` instance used in the example